google-beta/resource_project_service_identity_test.go

```diff
@@ -0,0 +1,35 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccProjectServiceIdentity_basic(t *testing.T) {
+	t.Parallel()
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckHealthcareDatasetDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testGoogleProjectServiceIdentity_basic(),
+			},
+		},
+	})
+}
+
+func testGoogleProjectServiceIdentity_basic() string {
+	return `
+resource "google_project_service_identity" "hc_sa" {
+	service = "healthcare.googleapis.com"
+}
+
+resource "google_project_iam_member" "hc_sa_bq_jobuser" {
+	project = google_project_service_identity.hc_sa.project
+	role    = "roles/bigquery.jobUser"
+	member  = "serviceAccount:service-${data.google_project.project.number}@gcp-sa-healthcare.iam.gserviceaccount.com"
```

Comment:
Can you add this datasource to the config as well? The project id should be filled in automatically, we should just need:

```hcl
data "google_project" "project" {}
```

Reply:
Done.

Reply:
Did you want me to make the example use this as well?

Reply:
Good catch. Yes please!

Reply:
Done.